app.py

```diff
@@ -14,6 +14,7 @@
 2) Send a processing request to the `MediaProcessor` C++ binary, which uses DeepFilterNet for fast filtering
 3) Serve the processed video on the frontend
 
+Refactored to improve maintainability and readability.
```

Comment:
Let's remove this, as it will quickly turn into a large changelog, which is not the best place for it.